vcr/assets/NutsAuthorizationCredential.config.yaml

```diff
@@ -0,0 +1,18 @@
+concept: authorization
+credentialType: NutsAuthorizationCredential
+indices:
+  - name: index_id
```

Comment:
do they need to be prefixed with `index_`, what if you don't?

Maybe a bit odd when you have to prefix every index name with `index_` (since in that case, the software should probably do it?)

Reply:
It translates to a sub-bucket on the collection. Only using the field as name would be weird as well. There are no requirements on the name.